hathor/websocket/factory.py

```diff
@@ -226,6 +239,9 @@ def send_or_enqueue(self, data):
         else:
             self.send_message(data)
 
+        # Send message to each connection that subscribed to it
+        self.execute_send(data, self.subscribed_connections[data['type']])
```

Comment:
There's an important difference here because when we subscribe to a specific message, there's no rate limit. I don't remember exactly why we implemented these limits, but should it also be considered here?

I'm more inclined to let the client deal with it. If they execute an expensive operation when receiving a ws message, they should do some sorte of rate limit on their side. E.g. an explorer should do batch updates on the screen in case there are too many txs arriving so the browser doesn't become unresponsive.

Reply:
We implemented the limits because the explorer and wallet UI was freezing when receiving many messages on the ws.

One of the reasons to do this refactor on the websocket was exactly because we don't want any throttle when receiving messages to update the wallet service. I think we should keep simple for now, i.e. all subscribed messages have no rate limit and we implement it in the future.

We already have the throttle implemented on the ws, so I believe we should allow the client to 'configure' the message subscription, like choosing if wants any throttle and each limit parameter. But we should discuss this in the future when decide to implement it.

Reply:
Agreed.